src/main/java/com/github/gilday/darkmode/DarkModeSync.java

```diff
@@ -50,7 +52,16 @@ public void dispose() {
 
   private void updateLafIfNecessary() {
     final LookAndFeelInfo current = lafManager.getCurrentLookAndFeel();
-    final boolean isDarkMode = isDarkMode();
+    boolean isDarkMode;
```

Comment:
Let's make this `final` to indicate that it must be set by the following conditional

Reply:
Done.